### PR TITLE
service protocol for access key

### PR DESCRIPTION
**Pull request by terencefan**
### Summary of the changes (Less than 80 chars)
 - Detail 1
 - Detail 2

Fixes #bugnumber (in this specific format)

### CODE REVIEW

File: src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs [hunk 1]
```diff
@@ -102,6 +102,117 @@ public interface IAckableMessage
...
 
+    /// <summary>
+    /// An access key request message.
+    /// </summary>
+    public class AccessKeyRequestMessage : ExtensibleServiceMessage
+    {
+        /// <summary>
+        /// Gets or sets the requested protocol version.
+        /// </summary>
+        public int Version { get; set; }
```
Comment (vwxyzh) [inline src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs:113]: remove

File: src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs [hunk 2]
```diff
@@ -102,6 +102,117 @@ public interface IAckableMessage
...
+        /// <summary>
+        /// Get or sets the key Id.
+        /// <c>null</c> 
+        /// </summary>
+        public string Kid { get; set; }
+
+        /// <summary>
+        /// Get or sets expire time in seconds.
+        /// </summary>
+        public int ExpireTime { get; set; }
```
Comment (vwxyzh) [inline src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs:129]: remove

File: src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs [hunk 3]
```diff
@@ -102,6 +102,117 @@ public interface IAckableMessage
...
+
+    /// <summary>
+    /// An access key response message.
+    /// </summary>
+    public class AccessKeyResponseMessage : ExtensibleServiceMessage
+    {
+        /// <summary>
+        /// Gets or sets the requested protocol version.
+        /// </summary>
+        public int Version { get; set; }
```
Comment (vwxyzh) [inline src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs:159]: remove

File: src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs [hunk 4]
```diff
@@ -102,6 +102,117 @@ public interface IAckableMessage
         int AckId { get; set; }
     }
 
+    /// <summary>
+    /// An access key request message.
+    /// </summary>
+    public class AccessKeyRequestMessage : ExtensibleServiceMessage
```
Comment (Y-Sindo) [inline src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs:108]: How about put them in a new class?

Comment (terencefan) [reply src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs:108]: All other messages are in the same class, I assume it's a coding convention.

Comment (wanlwanl) [reply src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs:108]: question: is this one being used in rest api? If so, I need a tracing id (impl `IMessageWithTracingId`) for request logs.

**Review by vwxyzh — APPROVED** [review]

**Pull request merged**